wp-content/themes/launchframe/assets/src/scss/_utilities.scss

```diff
@@ -1,3 +1,22 @@
 .clearfix, .cf, .clear {
     overflow: auto;
 }
+.u-bg {
+    &__color {
+        &--gray-dark {
+            background-color: $gray-dark;
+        }
+        &--gray {
+            background-color: $gray;
+        }
+        &--gray-light {
+            background-color: $gray-light;
+        }
+        &--gray-lighter {
+            background-color: $gray-lighter;
+        }
+        &--gray-lightest {
+            background-color: $gray-lightest;
+        }
+    }
+}
```

Comment:
.u refers to a utility. For consistency, I would either add it to your .font class or remove it from this .bg class